.github/workflows/ci.yml

```diff
@@ -44,9 +44,9 @@ jobs:
 
       - name: Run pytest with coverage
         run: |
-          pytest ./tests/addon -n 1 -m sequential --cov --cov-report=xml --retries 2
+          pytest ./tests/addon -n 0 -m sequential --cov --cov-report=xml --retries 2
           pytest ./tests/addon -n 4 -m "not sequential and not performance" --cov --cov-report=xml --cov-append --retries 2
-          pytest ./tests/addon -n 1 -m performance --cov-report=xml --cov-append --retries 2
+          pytest ./tests/addon -n 0 -m performance --cov-report=xml --cov-append --retries 2
```

Comment:
`-0` disabled `pytest-xdist` (parallelization) for the test and doesn't create an extra process unlike `-n 1`